app/src/main/java/com/T05/krowdtrialz/model/experiment/Experiment.java

```diff
@@ -97,6 +102,10 @@ public void setRegion(String region) {
         this.region = region;
     }
 
+    public boolean getStatus() {
```

Comment:
I think we should rename this to `isActive` so that the meaning of the result is clear.